src/web/src/components/ScheduleEvent.vue

```diff
@@ -1,8 +1,17 @@
 <template functional>
   <div class="schedule-event" :style="data.style">
     <div class="event-text">
-      {{ props.crn }} - {{ props.section }}
+      {{ props.title }}
       <br>
+      {{ props.name }} - {{ props.section }}
+      <br>
+      {{ props.crn }}
+      <br>
+      Location
```

Comment:
These should be removed completely. A comment out is fine, but we either want to display the actual location, or just not have a row.

Reply:
Forgot to push latest commit, should be updated on GitHub with them removed now.